### ISSUE TITLE
podman  generate kube without hostAliases

### ISSUE THREAD

**Issue by zhangguanzhang** [containers/podman#7468]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

/kind feature

**Description**

**Steps to reproduce the issue:**

1.create pod
```
podman pod create --name test \
  --add-host test.test.com:127.0.0.1 \
  --add-host test1.test.com:127.0.0.1
podman run -d --pod test --name test2  docker.io/library/nginx:alpine
```
2.  generate kube
```
podman  generate kube test
```

**Describe the results you received:**
the `spec.hostAliases` is null

**Describe the results you expected:**
```
  hostAliases:
  - hostnames:
    - test.test.com
    ip: 127.0.0.1
  - hostnames:
    - test1.test.com
    ip: 127.0.0.1
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
$ podman --version
podman version 2.0.5
```

**Output of `podman info --debug`:**

```
(paste your output here)
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
(paste your output here)
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**

Yes/No

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by bcstinch** [comment]
Does this fix #7442 as well? will hostAliases translate back into a podman pod from the kube yaml?

**Comment by zhangguanzhang** [comment]
after https://github.com/containers/podman/pull/7469 be merged, I will take a look at https://github.com/containers/podman/issues/7442

> Does this fix #7442 as well? will hostAliases translate back into a podman pod --add-host from the kube yaml?
